wrappers/golang/internal/generator/ecntt/templates/ecntt.h.tmpl

```diff
@@ -1,5 +1,4 @@
 #include <cuda_runtime.h>
-#include "../../../include/types.h"
```

Comment:
how go knows about NTTConfig and projective_t if there is nothing else included?

Reply:
They're partially declared lower in this file: https://github.com/ingonyama-zk/icicle/pull/481/files/2043ed13721067aba9e71ee381a41fd402a8958d#diff-ebd41d829ee4e442d0136366489b1f9d02faf670ac9553cdf7b7f21386bde13fR10-R11